mcs_benchmark_data/models/benchmark.py

```diff
@@ -2,6 +2,10 @@
 from dataclasses_json import LetterCase, dataclass_json
 from typing import NamedTuple, Tuple
 
+from rdflib import Graph
+from rdflib.resource import Resource
+from ..namespace import MCS, RDF

```

Comment:
I'd prefer absolute imports, makes it easier to move things around later if the need arises. So `mcs_benchmark_data.namespace`. You can change your IDE to prefer absolute imports on autocomplete.